### DISCUSSION TITLE
Broken line along tile boundary

### DISCUSSION THREAD

**Discussion by dkspec** [visgl/deck.gl#6459] [Q&A] [answered]
I have Path Layer that is rendered from MVT Layer.
When I zoom in, I noticed that some of the lines are cutoff along the tile boundaries.
Is there any props that im missing to prevent this from happening?
{
lineCapRounded: true,
jointRounded: true,
}

![image](https://user-images.githubusercontent.com/61771607/144310040-56021202-78bc-412f-bb2d-20f785aa3f2b.png)

Thank you!

**Comment by Pessimistress** [comment, marked as answer]
You need to configure a bigger buffer when you generate the MVT tiles. See https://github.com/visgl/deck.gl/issues/4549#issuecomment-622015243
